app/controllers/ruleset_controller.rb

```diff
@@ -36,7 +36,7 @@ def create
       if @ruleset_name == 'security_ruleset' # Call 42Crunch service
         @crunch_results = Linters::CrunchApi.new.lint_to_json(ruleset_params[:oas_file].tempfile)
       elsif @ruleset_name == 'government_ruleset' # Call Spectral service
-        @spectral_results = Linters::Spectral.new(file: ruleset_params[:oas_file]).lint_to_json
+        @spectral_results = JSON.parse(Linters::Spectral.new(file: ruleset_params[:oas_file]).lint_to_json)
```

Comment:
If parsing the JSON is to be used more generally, I'd suggest moving the parser to `Linters::Spectral`.

If the following was added to `Linters::Spectral`:
```ruby
def lint_to_objects
  JSON.parse(lint_to_json)
end
```
This method could be used when a parsed response is needed rather than the JSON string.

It would also then be easier to test this behaviour